modules/Bio/Otter/ServerAction/Region.pm

```diff
@@ -951,7 +947,7 @@ sub check_all_supporting_evidences {
     else {
       my $exon_toplevel = $toplevel_exon->transform('toplevel', $toplevel_exon->slice->coord_system->version);
       if ($exon_toplevel) {
-        $transcript->add_Exon($exon_toplevel);
+        $toplevel_exon = $exon_toplevel;
```

Comment:
I'm not sure this works because I don't think `$toplevel_exon` is part of the transcript. Maybe setting `$exon_toplevel` to `$exon`. But then you may have a problem with the if statement line 951.
Maybe `$exon = $exon_toplevel` would remove the need for the if statement line 951.